src/OpenTelemetry/Metrics/Exemplar/SimpleExemplarReservoir.cs

```diff
@@ -57,9 +57,13 @@ public override Exemplar[] Collect(ReadOnlyTagCollection actualTags, bool reset)
             if (reset)
```

Comment:
unrelated but kind of related nit: maybe this variable should be named `isDelta` for readability?

Reply:
yes... The Reservoir itself is not necessarily aware of temporality concept. All it needs to know is if it should reset its internal state or not.. The caller knows temporality and passes reset to this...

Reply:
Not a blocker for this PR, but just looking at this API shape, is the correct thing to do to expose `bool reset`?

> SDKs are free to decide whether "collect" should also reset internal storage for delta temporal aggregation collection, or use a more optimal implementation.

Curious what other languages have done to solve this.

Reply:
The current APIs need to be revised before exposing them to public (its marked internal), and there are many TODOs to be solved.